rye/src/bootstrap.rs

```diff
@@ -46,6 +46,7 @@ platformdirs==3.4.0
 pyproject_hooks==1.0.0
 requests==2.29.0
 tomli==2.0.1
+twine==4.0.2
```

Comment:
Since this adds a new dependency the `SELF_VERSION` needs bumping.

Reply:
Oh nice

Reply:
Added in e828154c1ecc35c197653477fdda0907642b85ad